Assets/BossRoom/Scripts/Client/Game/Entity/ClientBreakableVisualization.cs

```diff
@@ -36,7 +36,8 @@ public override void NetworkStart()
 
                 if (m_NetState.IsBroken.Value == true)
                 {
-                    //todo: don't dramatically break on entry to scene, if already broken.
+                    //A good future enhancement to BossRoom would be to add a "curtain" that hides transient loading effects.
```

Comment:
if pots are already broken when connecting, should either show pieces laying around or just not pot at all if it disappears. We shouldn't show the pot breaking dramatically.

Reply:
done.